### DISCUSSION TITLE
Fox Life | (Slovenia)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#58] [Show and tell]
### Channel Name

Fox Life

### Channel ID

FoxLifeSerbia.rs

### Stream URL

http://194.163.179.246/slovenci/filmski/foxlife.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
<img width="666" alt="image" src="https://user-images.githubusercontent.com/7253922/185268385-8e20b09c-0aa3-4faf-9aeb-6719b0e66ecb.png">
